data/deep missions.txt

```diff
@@ -0,0 +1,1366 @@
+			choice
+				`	"Alright, I'll do it."`
+					goto accept
+				`	"Sorry, but I'm not going to attack human beings for the sake of science."`
+					decline
+			
+			label friendly
+			`	"You told us that these humans were friendly. Do they allow you to purchase their technology?"`
+			branch license
+				has "license: Remnant"
+			`	"No," you respond. "They have not given me a license to buy any of their outfits or ships."`
+			`	The scientists ponder on this for a moment. Eventually, Maria speaks up. "Maybe if you assist them in some way they may give you access to their technology. Try doing jobs for them or see if anyone in their spaceports needs something from you."`
+			`	"You could also plunder their technology if they won't sell it to you," Ivan chimes in.`
+			choice
+				`	"I'll see if I can try to get my hands on what you need."`
+					goto accept
+				`	"Sorry, but I'm not interested in doing that kind of work for you."`
+					decline
+			
+			label license
+			`	"Yes," you respond. "I had to do a few odd jobs for them, but they eventually gave me license to purchase their outfits and ships."`
+			`	"Then all you need to do is buy whatever we ask you to and bring it back here. Are you interested?"`
+			choice
+				`	"I'd be glad to help"`
+				`	"Sorry, but I don't want to spend money for your research."`
+					decline
+			
+			label accept
+			`	"Great!" Ivan exclaims. "Right now we'd be interested in studying their engines. Please bring us one Forge-Class Thruster and one Forge-Class Steering. They shouldn't be too hard to get."`
+				accept
+	
+	on decline
+		set "Deep: Remnant Research: done"
+			
+	on visit
+		dialog `You have landed on <planet>, but you do not have both a Forge-Class Thruster and Forge-Class Steering. Return to <planet> once you have both engines.`
+		
+	on complete
+		payment 417000
+		outfit "Forge-Class Thruster" -1
+		outfit "Forge-Class Steering" -1
+
+
+
+mission "Deep: Remnant: Generators"
+	landing
+	name `Fetching Generators`
+	description `Acquire a Millennium Cell from the Remnant and bring the generator to <destination>.`
+	source Valhalla
+	to offer
+		has "Deep: Remnant: Engines: done"
+	on offer
+		conversation
+			`The scientists are able to quickly have the engines removed from your ship and ready in their research lab thanks to it being much closer to the spaceport. They pay you 417,000 credits for bringing the engines to them. After about three hours of analyzing the engines, Ivan walks up to you.`
+			`	"This is very interesting tech. They seem similar to human ion engines, but surprisingly better than even atomic engines, being both more powerful and more efficient at the same time. They're slightly more power-hungry, but it's still amazing that such an isolated group of humans could make such a technological leap in such a short amount of time."`
```

Comment:
Change `it's still amazing that such an isolated group of humans could make such a technological leap in such a short amount of time.` to `it's amazing that such an isolated group of humans could make technological leaps beyond what the Deep has done.`?